src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const sum = {
+    result: 0,
+
+    add(number) {
+      sum.result += number;
+    },
+
+    subtract(number) {
+      sum.result -= number;
+    },
+
+    multiply(number) {
+      sum.result *= number;
+    },
+
+    divide(number) {
+      sum.result /= number;
+    },
```

Comment:
You need to add a check by 0, because according to the rules of mathematics, you cannot divide by 0